Filter/StringFilter.php

```diff
@@ -25,28 +25,39 @@ public function filter(ProxyQueryInterface $proxyQuery, $alias, $field, $data)
             return;
         }
 
-        $data['value'] = trim($data['value']);
+        $value = $data['value'] = trim($data['value']);
```

Comment:
cause you can change the filters, right? I don't ever use the array value again, as i use a local var.